README.md

```diff
@@ -117,23 +111,21 @@ Our [prioritized roadmap][roadmap] of features and utilities that the core team
 
 We'd like to directly mention (in alphabetical order) for their continued community support this month and helping directly make PowerToys a better piece of software.  
 
-[@Aaron-Junker](https://github.com/Aaron-Junker), 
-[@davidegiacometti](https://github.com/davidegiacometti), 
-[@Drakula44](https://github.com/Drakula44), 
-[@htcfreek](https://github.com/htcfreek),
-[@Jay-o-Way](https://github.com/Jay-o-Way),
-[@niels9001](https://github.com/niels9001),
-and 
-[@notDevagya](https://github.com/notDevagya)
+[@BenConstable9](https://github.com/BenConstable9), 
```

Comment:
Is there a reason why I got removed from the list?

Reply:
we're not done.  Deondre did a first pass for me, i'm going through right now and adjusting